File: day 55/homework.js
document.getElementById("mathForm").addEventListener("submit", function(event){
    event.preventDefault()
    var num1 = document.getElementById("num1").value
    var num2 = document.getElementById("num2").value
    console.log("First Number:", num1)
    console.log("Second Number:", num2)
})

function displayText() {
    var text = document.getElementById("textInput").value
    var color = document.getElementById("colorInput").value
    var paragraph = document.createElement("p")
    paragraph.style.color = color
    paragraph.textContent = text
    document.body.appendChild(paragraph)
}

File: day 55/homework.py
num1 = float(input("Enter the first number: "))
num2 = float(input("Enter the second number: "))

# Addition
addition = num1 + num2
print("Addition Result:", addition)

# Subtraction
subtraction = num1 - num2
print("Subtraction Result:", subtraction)

# Multiplication
multiplication = num1 * num2
print("Multiplication Result:", multiplication)

# Division
if num2 != 0:
    division = num1 / num2
    print("Division Result:", division)
else:
    print("Cannot divide by zero.")